body/test/README.md

```diff
@@ -0,0 +1,4 @@
+Unit tests can be run by, for example:
```

Comment:
This is already covered [here](https://github.com/hello-robot/stretch_body/tree/master/body#running-tests).